meson.build

```diff
@@ -4,7 +4,8 @@ project(
   license : 'GPL-3.0-or-later',
   default_options : [
     'c_std=c99',
-    'warning_level=3'
+    'warning_level=3',
+    'prefix=/usr'
```

Comment:
We should not alter the default value of `prefix`. This is usually set by package maintainers or distributions anyway and is not something we should care about in partup itself.